src/pages/seongjisue/Login/Login.js

```diff
@@ -1,9 +1,68 @@
-import React from 'react';
+import React, { Component } from 'react';
+import './Login.scss';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      buttonColor: false,
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value });
+    this.changeButtonColor();
```

Comment:
이렇게 하면 순서 보장이 되지 않기 때문에 아래처럼 해주셔야 id와 pw가 업데이트 되고 난 이후 정확히 비교할 수 있습니다
```suggestion
    this.setState({ [name]: value }, this.changeButtonColor);
```